core/src/main/java/cucumber/runner/OrderedTestRunEventBus.java

```diff
@@ -23,14 +33,62 @@ public Long getTime() {
     public void send(final Event event) {
         super.send(event);
         queue(event);
-    }
 
-    private void queue(final Event event) {
-        queue.add(event);
         if (event instanceof TestRunFinished) {
```

Comment:
this will result in nothing being sent to the `ProgressFormatter` until the end of the run.

Reply:
Yup. That's why this is in the wrong spot:

https://github.com/cucumber/cucumber-jvm/pull/1389/files#diff-6ec644ece6e92b60fe39b4cceef33c08R199